ci-operator/config/openshift/vertical-pod-autoscaler-operator/openshift-vertical-pod-autoscaler-operator-master.yaml

```diff
@@ -1,4 +1,8 @@
 base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.59.1
```

Comment:
Do you know, will this auto-bump when they cut new channels, or are we going to have to police this? ( I just know if we skew too far from the golang version weird stuff happens)

Reply:
I think this would have to be policied by us. It should probably match whatever is in our `Makefile`'s version. So when we do operator-sdk upgrades (by release chore cycles or however we want to do it), we should change it here as well so nothing unexpected happens.